localisation/Workbench v1.0/en_IN/rainmaker-workbench

```diff
@@ -0,0 +1,614 @@
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "P G R. Service Defs",
+        "message": "PGR Service Defs",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_DEPARTMENT",
+        "message": "Department",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_DESCRIPTION",
+        "message": "Description",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_FROMDATE",
+        "message": "From Date",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_ID",
+        "message": "Service Def Id",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_SERVICECODE",
+        "message": "Service Code",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_SERVICEDEFSCATEGORY",
+        "message": "Service Def Category",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_SLAHOURS",
+        "message": "SLA Hours",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "PGR_SERVICEDEFS_TODATE",
+        "message": "To Date",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "SCHEMA_COMMON_MASTERS_UOM",
+        "message": "Common Masters UOM",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "SCHEMA_PGR_SERVICEDEFS",
+        "message": "PGR Service Defs",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_ADD_MDMS",
+        "message": "Add Master data",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_ADD_MDMS_ADD_ACTION",
+        "message": "Add data",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_EDIT_MDMS",
+        "message": "Edit Master Data",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_ERROR_MDMS_DATA",
+        "message": "Error : Error",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_FIELD",
+        "message": "Field",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_FIELD_VALUE",
+        "message": "Value",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
+        "code": "WBH_MASTER_NAME",
+        "message": "Master Name",
+        "module": "rainmaker-workbench",
+        "locale": "en_IN"
+    },
+    {
```

Comment:
`{
        "code": "WBH_MDMS_COMMON_MASTERS_DEPARTMENT",
        "message": "Department",
        "module": "rainmaker-workbench",
        "locale": "en_IN"
    },
    {
        "code": "WBH_MDMS_COMMON_MASTERS_UOM",
        "message": "Unit of Measurement",
        "module": "rainmaker-workbench",
        "locale": "en_IN"
    },
    {
        "code": "WBH_MDMS_MASTER_COMMON_MASTERS",
        "message": "Common Masters",
        "module": "rainmaker-workbench",
        "locale": "en_IN"
    },
    {
        "code": "WBH_MDMS_MASTER_PGR",
        "message": "PGR",
        "module": "rainmaker-workbench",
        "locale": "en_IN"
    },
    {
        "code": "WBH_MDMS_PGR_SERVICEDEFS",
        "message": "PGR Service Defs",
        "module": "rainmaker-workbench",
        "locale": "en_IN"
    },
    `
    
    move these schema related localisation to 
schema module